### DISCUSSION TITLE
Add time on the current playing sound pop-up

### DISCUSSION THREAD

**Discussion by Corne2Plum3** [ppy/osu#20241] [Ideas]
It would be nice if you add time on the current playing sound pop-up, for example doing something like this:

![osu_2022-09-11_14-34-04_concept](https://user-images.githubusercontent.com/57914482/189528227-2b89e69a-f251-4c01-8b35-fc1de9835fe9.png)

It's a very little thing, but I think it would be a great addition.

**Comment by A47C** [comment]
I agree!

**Comment by Voider404** [comment]
Agreed
